### ISSUE TITLE
Regions go to wrong location when switching files

### ISSUE THREAD

**Issue by keflavich** [CARTAvis/carta-frontend#1493]
I have a situation where two overlapping, but not identical images are open in CARTA.  I loaded some regions, then switched between the two images.  If the images were not XY-matched, the regions show up in the wrong place in the second image.  If they are XY-matched, they show up in the correct place.  I haven't tested how to reproduce this with a MWE yet, though.

**Comment by veggiesaurus** [comment]
if the two images are not matched, they do not share region sets. Are you loading the regions in each image separately?

**Comment by keflavich** [comment]
No.  I opened a region file in one image, then switched to the other, but I did not separately load the regions.

It may be that I had them XY-matched, then XY-unmatched, then switched?  I'm not really sure I can reproduce the exact order of operations, but I am reasonably sure I will cause this issue again.

**Comment by veggiesaurus** [comment]
> It may be that I had them XY-matched, then XY-unmatched, then switched? I'm not really sure I can reproduce the exact order of operations, but I am reasonably sure I will cause this issue again.

Ah ok, that makes sense. If you do the following:
1. Load an image
2. Load a second image and match them
3. Load regions (this will be shared across both images)
4. Unmatch Image 2
5. Move the regions around in Image 2

Then you'll find that the regions in Image 1 and 2 have diverged.

**Comment by kswang1029** [comment]
Is this still an issue?

**Comment by YuHsuan-Hwang** [comment]
We can reopen this issue when there is further information.
